frontend/src/app/new-claim/new-claim.component.ts

```diff
@@ -66,12 +65,12 @@ export class NewClaimComponent implements OnInit {
   }
 
   public alphaNumericValidator(event: any) {
-    const charactersAllowed = /^[a-zA-Z0-9]*$/;   
+    const charactersAllowed = /^[a-zA-Z0-9]*$/;
```

Comment:
Was this just prettier or did you actually find this extra space

Reply:
You'll never know :smiling_imp: